apps/pie-docs/src/patterns/cookie-banner/code/code.md

```diff
@@ -0,0 +1,161 @@
+    events: "{% include './events.json' %}"
+---
+
+## Overview
+
+<p>
+  <a href="https://www.npmjs.com/@justeattakeaway/pie-cookie-banner">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-cookie-banner.svg">
+  </a>
+</p>
+
+`pie-cookie-banner` is a Web Component built using the Lit library.
+
+This component can be easily integrated into various frontend frameworks and customized through a set of properties.
+
+## Cookies
+`pie-cookie-banner` purely handles the UI and user interactions for cookie consent. It does not handle the setting or management of cookies or the storage of user preferences. It is the responsibility of the consuming application to handle these aspects. See [Events](http://localhost:8080/patterns/cookie-banner/code/#events) for more information.
+
+
+## Installation
+
+To install `pie-cookie-banner` in your application, run the following on your command line:
+
+```shell
+# npm
+$ npm i @justeattakeaway/pie-cookie-banner
+```
+
+```shell
+# yarn
+$ yarn add @justeattakeaway/pie-cookie-banner
+```
+
+{% notification {
+  type: "neutral",
+  iconName: "link",
+  message: "For more information on using PIE components as part of an application, check out the [Getting Started Guide.](https://github.com/justeattakeaway/pie/wiki/Getting-started-with-PIE-Web-Components)."
+} %}
+
+## Playground
+
+ <iframe
+  src="https://webc.pie.design/?path=/story/cookie-banner--default&viewMode=story&shortcuts=true&singleStory=true"
+  width="100%"
+  height="600px"
+  style="border: none; margin-top: 32px;"
+></iframe>
+
+
+## Props
+
+{% componentDetailsTable {
+  tableData: props
+} %}
+
+## Events
+
+The pie cookie banner does not directly establish cookies within the application or component. Instead, it offers events that application developers (consumers of this component) can subscribe to. This, in turn, enables applications to manage the process of setting their own cookies. The table below provides a list of events you can listen to:
+
+{% componentDetailsTable {
+tableData: events
+} %}
+
+## Localisation
+
+By default, the component displays its content in English language. To display the content in another language, you need to import the locale data for that language and pass it in the `locale` prop. For example, to display the content in Dutch, you need to import the Dutch locale data:
+
+```js
+import locale from '@justeattakeaway/pie-cookie-banner/locales/nl-nl.json';
+
+<!-- JSX -->
+<PieCookieBanner locale={locale}></PieCookieBanner>
+```
+
+It's possible to import all locales at once, if necessary:
+
+```js
+import allLocales from '@justeattakeaway/pie-cookie-banner/locales';
+```
+
+## Examples
+
+For HTML:
+
+```js
+// import as module into a js file e.g. main.js
+import '@justeattakeaway/pie-cookie-banner'
+```
+
+```html
+<!-- pass js file into <script> tag -->
+<pie-button type="reset" isFullWidth="true" onclick="e => console.log(e)">Click me!</pie-button>
```

Comment:
Should this be a cookie banner example?

Reply:
😄 I am trying to remember why I wanted to include this examples, I went back and forth and forgot to update this one. I will update the tag, good catch! let me know if you think is better to remove that particular example.